target/linux/ramips/image/mt76x8.mk

```diff
@@ -213,6 +213,24 @@ define Device/tl-wr841n-v13
 endef
 TARGET_DEVICES += tl-wr841n-v13
 
+define Device/tl-wr841n-v14
```

Comment:
Please include `Device/tplink` here instead of defining the very same values.

Reply:
The tftp-recovery image differs between the Device/tplink target and the wr841-v14 target.
Original with 128k padding:
`IMAGE/tftp-recovery.bin := pad-extra 128k | $$(IMAGE/factory.bin)`
WR841-v14 with 64k padding:
`IMAGE/tftp-recovery.bin := pad-extra 64k | $$(IMAGE/factory.bin)`

Should I nevertheless include the Device/tplink and just keep the one changed line to override the default value?

Reply:
@donothingloop One more thing I've just noticed `Device/tplink_tl-wr841n-v14` in order to get proper `SUPPORTED_DEVICES` and thus working sysugpgrade support.

Reply:
@donothingloop You need to change `Device/tl-wr841n-v14` to `Device/tplink_tl-wr841n-v14`